components/MenuList/ListingItems/ListingItems.tsx

```diff
@@ -20,15 +20,17 @@ export default function Listing({ item, isPromotionActive }: Props) {
       })
    }, [])
 
+   const activation = activeItem&&item.isAvailable
```

Comment:
Aren't you getting an error on this line? There is no space

Reply:
Deu erro não, mas vou alterar mesmo assim.

Reply:
No error found but I'll change it anyway.